src/mmw/js/src/core/utils.js

```diff
@@ -96,7 +96,7 @@ var utils = {
     },
 
     noDataSort: function(x, y) {
-        var noData = 'no data';
+        var noData = 'No Data';
```

Comment:
Wonder if it's worth making 'No Data' into a global constant to make it easier to change in the future.